<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindowClass</class>
 <widget class="QDialog" name="SettingsWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="filesGroup">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Default directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="defaultFolderInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseFolderButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelEncryption_2">
          <property name="text">
           <string>Listening port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="listenPortSpinBox">
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>50000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelEncryption_3">
          <property name="text">
           <string>Max connections</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxConnectionsSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="labelEncryption_4">
          <property name="text">
           <string>Max download KB/s</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxDlSpinBox">
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="labelEncryption_5">
          <property name="text">
           <string>Max upload KB/s</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxUpSpinBox">
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="upnpCheckBox">
        <property name="text">
         <string>UPNP port mapping</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="upnpInfoLabel">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Protocol</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="dhtCheckBox">
        <property name="text">
         <string>DHT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dhtInfoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="utpCheckBox">
        <property name="text">
         <string>uTP protocol support</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="labelEncryption">
          <property name="text">
           <string>Protocol encryption</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="encryptionComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="associateButton">
     <property name="text">
      <string>Associate with magnet links</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QDialogButtonBox" name="buttonsBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
